package/MDAnalysis/converters/OpenMMParser.py

```diff
@@ -187,23 +181,30 @@ def _mda_topology_from_omm_topology(self, omm_topology):
                     warnings.warn("For absent elements, atomtype has been  "
                                   "set to 'X' and mass has been set to 0.0. "
                                   "If needed these can be guessed using "
-                                  "MDAnalysis.topology.guessers.")
+                                  "universe.guess_TopologyAttributes(to_guess=['masses', 'types']). "
+                                  "(for MDAnalysis version 2.1 this is done automatically, but it will be removed in "
+                                  "future vesrions).")
                 attrs.append(Elements(np.array(validated_elements,
                                                dtype=object)))
+                attrs.append(Atomtypes(np.array(atomtypes, dtype=object)))
+                attrs.append(Masses(np.array(masses)))
 
             else:
-                atomtypes = guess_types(atomnames)
-                masses = guess_masses(atomtypes)
                 wmsg = ("Element information is missing for all the atoms. "
                         "Elements attribute will not be populated. "
                         "Atomtype attribute will be guessed using atom "
                         "name and mass will be guessed using atomtype."
-                        "See MDAnalysis.topology.guessers.")
+                        "for MDAnalysis version 2.1 this is done automatically, but it will be removed in "
+                        "future vesrions. "
```

Comment:
```suggestion
                        "future versions. "
```
Which? We need to be clear about when this break happens - my suggestion is 3.0 (it's a big behaviour change that will affect results)